### ISSUE TITLE
Add shouldIanswer as screener

### ISSUE THREAD

**Issue by jrotaetxe** [thess/callattendant#22]
Hello all.
Based on the model proposed by Cloud_forge
https://github.com/emxsys/callattendant/pull/146
I have been using this different screening service for some time, but suddenly it has stopped and is returning a "403  Forbidden". This service is servicing european numbers and is the reference for these countries.
Could any aof you, with better knowledge check what is going on with this?
Thank you in advance

**Comment by thess** [comment]
I looked at this some time ago and put it aside due to the failure you mention. I also noted the reference to the French SIA site and not the general international version. Now... I have updated the module to function correctly (at least in the US where I can test it). It is not yet integrated into Callattendant but I pushed a working version of the module `shouldianswer.py` here if you want to try it in a version that has it supported. 1ed09a3 IMMV.
